### PR TITLE
add param type and forcenew in resource docs

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**

### CODE REVIEW

File: docs/resources/blockstorage_volume_v2.md [hunk 1]
```diff
@@ -23,42 +23,41 @@ resource "huaweicloud_blockstorage_volume_v2" "volume_1" {
...
-* `source_vol_id` - (Optional) The volume ID from which to create the volume.
+* `source_vol_id` - (Optional, String, ForceNew) The volume ID from which to create the volume.
     Changing this creates a new volume.
 
-* `volume_type` - (Optional) The type of volume to create. Available types are
+* `volume_type` - (Optional, String, ForceNew) The type of volume to create. Available types are
     `SSD`, `SAS` and `SATA`. Changing this creates a new volume.
 
-* `cascade` - (Optional, Default:false) Specifies to delete all snapshots associated with the EVS disk.
+* `cascade` - (Optional, Bool, Default:false) Specifies to delete all snapshots associated with the EVS disk.
```
Comment (niuzhenguo) [inline docs/resources/blockstorage_volume_v2.md]: do we need to set Default:false here?

File: docs/resources/compute_instance.md [hunk 2]
```diff
@@ -187,104 +187,104 @@ function, or the `template_cloudinit_config` resource.
...
 
-* `region` - (Optional) The region in which to create the instance. If omitted,
+* `region` - (Optional, String, ForceNew) The region in which to create the instance. If omitted,
 	the provider-level region will be used. Changing this creates a new server.
 
-* `name` - (Required) A unique name for the resource.
+* `name` - (Required, String) A unique name for the resource.
 
-* `image_id` - (Optional; Required if `image_name` is empty) The image ID of
+* `image_id` - (Optional, String; Required if `image_name` is empty) The image ID of
```
Comment (niuzhenguo) [inline docs/resources/compute_instance.md]: More `Required ..` out from ()

File: docs/resources/fw_firewall_group_v2.md [hunk 3]
```diff
@@ -47,36 +47,36 @@ resource "huaweicloud_fw_firewall_group_v2" "firewall_group_1" {
...
     to create a firewall group for another tenant. Changing this creates a new
     firewall group.
 
-* `ports` - (Optional) Port(s) to associate this firewall group instance
+* `ports` - (Optional, String) Port(s) to associate this firewall group instance
     with. Must be a list of strings. Changing this updates the associated routers
     of an existing firewall group.
 
-* `value_specs` - (Optional) Map of additional options.
+* `value_specs` - (Optional, Map, ForceNew, String) Map of additional options.
```
Comment (niuzhenguo) [inline docs/resources/fw_firewall_group_v2.md]: duplicate String

File: docs/resources/fw_policy_v2.md [hunk 4]
```diff
@@ -42,38 +42,38 @@ resource "huaweicloud_fw_policy_v2" "policy_1" {
...
     `shared` status of an existing firewall policy. Only administrative users
     can specify if the policy should be shared.
 
-* `tenant_id` - (Optional) The owner of the firewall policy. Required if admin wants
+* `tenant_id` - (Optional, String, ForceNew) The owner of the firewall policy. Required if admin wants
     to create a firewall policy for another tenant. Changing this creates a new
     firewall policy.
 
-* `value_specs` - (Optional) Map of additional options.
+* `value_specs` - (Optional, Map, ForceNew, String) Map of additional options.
```
Comment (niuzhenguo) [inline docs/resources/fw_policy_v2.md]: duplicate String

File: docs/resources/nat_dnat_rule.md [hunk 5]
```diff
@@ -26,31 +26,31 @@ resource "huaweicloud_nat_dnat_rule" "dnat_1" {
...
+* `nat_gateway_id` - (Required, String, ForceNew) ID of the nat gateway this dnat rule belongs to.
    Changing this creates a new dnat rule.
 
-* `port_id` - (Optional) Specifies the port ID of an ECS or a BMS.
+* `port_id` - (Optional, String, ForceNew) Specifies the port ID of an ECS or a BMS.
   This parameter and private_ip are alternative. Changing this creates a
   new dnat rule.
 
-* `private_ip` - (Optional) Specifies the private IP address of a
+* `private_ip` - (Optional, ForceNew, String, ForceNew) Specifies the private IP address of a
```
Comment (niuzhenguo) [inline docs/resources/nat_dnat_rule.md]: duplicate ForceNew

File: docs/resources/networking_port.md [hunk 6]
```diff
@@ -25,78 +25,78 @@ resource "huaweicloud_networking_port" "myport" {
...
-* `tenant_id` - (Optional) The owner of the Port. Required if admin wants
+* `tenant_id` - (Optional, String, ForceNew) The owner of the Port. Required if admin wants
     to create a port for another tenant. Changing this creates a new port.
 
-* `device_owner` - (Optional) The device owner of the Port. Changing this creates
+* `device_owner` - (Optional, String, ForceNew) The device owner of the Port. Changing this creates
     a new port.
 
-* `security_group_ids` - (Optional - Conflicts with `no_security_groups`) A list
+* `security_group_ids` - (Optional, List; Conflicts with `no_security_groups`) A list
```
Comment (niuzhenguo) [inline docs/resources/networking_port.md]: Move `Conflicts with ...` out from ()

File: docs/resources/networking_port.md [hunk 7]
```diff
@@ -25,78 +25,78 @@ resource "huaweicloud_networking_port" "myport" {
...
     of security group IDs to apply to the port. The security groups must be
     specified by ID and not name (as opposed to how they are configured with
     the Compute Instance).
 
-* `no_security_groups` - (Optional - Conflicts with `security_group_ids`) If set to
+* `no_security_groups` - (Optional, List; Conflicts with `security_group_ids`) If set to
```
Comment (niuzhenguo) [inline docs/resources/networking_port.md]: ditto

File: docs/resources/obs_bucket.md [hunk 8]
```diff
@@ -143,60 +143,60 @@ resource "huaweicloud_obs_bucket" "bucket" {
...
-* `target_bucket` - (Required) The name of the bucket that will receive the log objects.
+* `target_bucket` - (Required, String) The name of the bucket that will receive the log objects.
   The acl policy of the target bucket should be `log-delivery-write`.
-* `target_prefix` - (Optional) To specify a key prefix for log objects.
+* `target_prefix` - (Optional, String) To specify a key prefix for log objects.
 
 The `website` object supports the following:
 
-* `index_document` - (Required, unless using `redirect_all_requests_to`) Specifies the default homepage of the static website, only HTML web pages are supported.
+* `index_document` - (Required, String; unless using `redirect_all_requests_to`) Specifies the default homepage of the static website, only HTML web pages are supported.
```
Comment (niuzhenguo) [inline docs/resources/obs_bucket.md]: Move `unless using...` out from ()

File: docs/resources/sfs_file_system.md [hunk 9]
```diff
@@ -31,33 +31,33 @@ resource "huaweicloud_sfs_file_system" "share-file" {
...
+* `name` - (Optional, String) The name of the shared file system.
 
-* `description` - (Optional) Describes the shared file system.
+* `description` - (Optional, String) Describes the shared file system.
 
-* `is_public` - (Optional) The level of visibility for the shared file system.
+* `is_public` - (Optional, Bool, ForceNew) The level of visibility for the shared file system.
 
-* `metadata` - (Optional) Metadata key and value pairs as a dictionary of strings. Changing this will create a new resource.
+* `metadata` - (Optional, Map, ForceNew, String) Metadata key and value pairs as a dictionary of strings. Changing this will create a new resource.
```
Comment (niuzhenguo) [inline docs/resources/sfs_file_system.md]: duplicate String

File: docs/resources/blockstorage_volume_v2.md (hunk 1 above)
Comment (Jason-Zhang9309) [reply docs/resources/blockstorage_volume_v2.md]: fixed

File: docs/resources/compute_instance.md (hunk 2 above)
Comment (Jason-Zhang9309) [reply docs/resources/compute_instance.md]: fixed

File: docs/resources/fw_firewall_group_v2.md (hunk 3 above)
Comment (Jason-Zhang9309) [reply docs/resources/fw_firewall_group_v2.md]: fixed

File: docs/resources/fw_policy_v2.md (hunk 4 above)
Comment (Jason-Zhang9309) [reply docs/resources/fw_policy_v2.md]: fixed

File: docs/resources/nat_dnat_rule.md (hunk 5 above)
Comment (Jason-Zhang9309) [reply docs/resources/nat_dnat_rule.md]: fixed

File: docs/resources/networking_port.md (hunk 6 above)
Comment (Jason-Zhang9309) [reply docs/resources/networking_port.md]: fixed

File: docs/resources/networking_port.md (hunk 7 above)
Comment (Jason-Zhang9309) [reply docs/resources/networking_port.md]: fixed

**Pull request merged**